features/frameworks/greentea-client/source/test_env.cpp

```diff
@@ -574,7 +574,6 @@ int greentea_parse_kv(char *out_key,
             break;
         }
     }
-    return 0;
 }
```

Comment:
Do you not then get a warning for a function requiring a return value not supplying one ?